src/tokens/solana.tokenlist.json

```diff
@@ -104874,6 +104874,19 @@
         "website": "https://theacesnft.com/"
       }
     },
+    {
+      "chainId": 101,
+      "address": "8HdsVCvd7fTrLVuwiFnkXiwCC1PjgxaqbgKf66dHmPJU",
+      "symbol": "MMMEME",
+      "name": "MMMEME TOKEN",
+      "decimals": 6,
+      "logoURI": "C:\Users\elesh\OneDrive\Documents\GitHub\token-list\assets\mainnet\8HdsVCvd7fTrLVuwiFnkXiwCC1PjgxaqbgKf66dHmPJU\mmmeme.png",
```

Comment:
this wont work , you should use `https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/8HdsVCvd7fTrLVuwiFnkXiwCC1PjgxaqbgKf66dHmPJU\mmmeme.png`